metadata/coanzse.org.xml

```diff
@@ -0,0 +1,191 @@
+         </saml:Attribute>
+      </mdattr:EntityAttributes>
+   </md:Extensions>
+   <md:SPSSODescriptor protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol">
+      <md:Extensions>
+         <mdui:UIInfo>
+            <mdui:DisplayName xml:lang="en">CoANZSE Audio</mdui:DisplayName>
+            <mdui:Description xml:lang="en">CoANZSE Audio Description</mdui:Description>
+            <mdui:InformationURL xml:lang="en">https://coanzse.org</mdui:InformationURL>
+            <mdui:Logo height="220" width="195">https://coanzse.org/corpus-frontend-3.1.0/test/static/img/output-onlinepngtools.png</mdui:Logo>
+            <mdui:Keywords xml:lang="en">Corpus of Australian and New Zealand Spoken English+English+Audio+YouTube Transcripts</mdui:Keywords>
+            <mdui:PrivacyStatementURL xml:lang="en">https://catalog.clarin.eu/privacy_statement.xhtml</mdui:PrivacyStatementURL>
+         </mdui:UIInfo>
+         <init:RequestInitiator Binding="urn:oasis:names:tc:SAML:profiles:SSO:request-init"
+                                Location="https://coanzse.org/Shibboleth.sso/Login"/>
+         <idpdisc:DiscoveryResponse Binding="urn:oasis:names:tc:SAML:profiles:SSO:idp-discovery-protocol"
+                                    Location="https://coanzse.org/Shibboleth.sso/Login"
+                                    index="1"/>
+      </md:Extensions>
+      <md:KeyDescriptor>
+         <ds:KeyInfo>
+            <ds:X509Data>
+               <ds:X509Certificate>
+MIIEGTCCAwGgAwIBAgISA9kUpaXkAnr/cOZVJJxAUhnwMA0GCSqGSIb3DQEBCwUA
+MDIxCzAJBgNVBAYTAlVTMRYwFAYDVQQKEw1MZXQncyBFbmNyeXB0MQswCQYDVQQD
+EwJSMzAeFw0yMzA4MjExNDI0NDNaFw0yMzExMTkxNDI0NDJaMBYxFDASBgNVBAMT
+C2NvYW56c2Uub3JnMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEvZ13BXoUVHCs
+0D5WGwMiKf++kq6j+HcYSNk6wK0B3zHzQm+93sxGdSKkTif30lUvxayKC3XxoUOE
+BrGZNnBMnKOCAg4wggIKMA4GA1UdDwEB/wQEAwIHgDAdBgNVHSUEFjAUBggrBgEF
+BQcDAQYIKwYBBQUHAwIwDAYDVR0TAQH/BAIwADAdBgNVHQ4EFgQUkDmjJVAEbRvq
+oHkPVb4XFeW4/lAwHwYDVR0jBBgwFoAUFC6zF7dYVsuuUAlA5h+vnYsUwsYwVQYI
+KwYBBQUHAQEESTBHMCEGCCsGAQUFBzABhhVodHRwOi8vcjMuby5sZW5jci5vcmcw
+IgYIKwYBBQUHMAKGFmh0dHA6Ly9yMy5pLmxlbmNyLm9yZy8wFgYDVR0RBA8wDYIL
+Y29hbnpzZS5vcmcwEwYDVR0gBAwwCjAIBgZngQwBAgEwggEFBgorBgEEAdZ5AgQC
+BIH2BIHzAPEAdwC3Pvsk35xNunXyOcW6WPRsXfxCz3qfNcSeHQmBJe20mQAAAYoY
+ssZaAAAEAwBIMEYCIQDB+oD1HS77NEVAub6vB5nKFPL5jCweBlaMakdbU0QwhQIh
+AL99yrl8ZAfvJymOc9AqkMJNGxhdNHrYfzpRKB3+w+uHAHYAejKMVNi3LbYg6jjg
+Uh7phBZwMhOFTTvSK8E6V6NS61IAAAGKGLLGcAAABAMARzBFAiBez/XvF8WNv5gx
+7JZEFkR/EFlJXIKtpvZAv5PnywvF6QIhAO0hclOw7Tkwgugi2G5KpR1kcdeoLPWj
+vsFByac8JttaMA0GCSqGSIb3DQEBCwUAA4IBAQABy8EtdpT1MMbh6j5lLP5cHL+c
+XwgzxZqDXB6tZ8L55bTmQHGwBQRs83mcEp9x0qNq1E/s4fZPpe/IWagO+CNLbbes
+py4I1XtG+wP/HQfzvGO2TAIGT5l+5WqPHqx9328zGmLGcWHOt7rrwNTVduWBuw+H
++xHSjj0kMpDlea6Jc6o7zaQKFEFKXQeUd6vZITax6vAblgiIzJZJmQK2r0Pu/HEI
+SRxyyg7vlfUdhis07KK7snpJeoQzlWxVqx/9RXE+wAezPrkEDWVmD6oOlREcZyRe
+8ferSAK/gNtsv1iZswEOlCmo2Yk3TT4GfxPIW8rz4Qo1Y1zdELpcV6m0eRi8
+</ds:X509Certificate>
+<ds:X509Certificate>
```

Comment:
There are currently 3 certificates in this file and only one is expected.
We advice you use the same certificate for both: signing and encryption. This certificate should be self-signed (currently it is a letsencrypt signed certificate) and have a validity of 4-5 years.